### DISCUSSION TITLE
Request to help

### DISCUSSION THREAD

**Discussion by Zerihun0848** [jddingemanse/awtiCodeDev#24] [Q&A]
Dear Johannes Dirk Dingemanse and Nimrod,
-  I would like to say thank your for nice initiation to support us.  

Request to help
- Please show simple steps in NTCD file to merge, average,. max, min etc.
- Where I get python code ?
- How to connect python with Machin learning and 
- Generally NTCD file

**Comment by jddingemanse** [comment]
Hello Zerihun,

Can you explain what an NTCD file is? Or do you mean a NetCDF (.nc) file? If the latter, you can check materials for Module 5.1 of the [training](https://github.com/jddingemanse/awtiCodeDev/tree/main/training).

As for machine learning: that is a relevant and way to big topic to simply answer. Simple answer: make sure you practice a lot with Python, upon which you will be able to understand some basic tutorials and relevant packages that you can find online.
